tests/nested/manual/uc20-create-partitions-encrypt/task.yaml

```diff
@@ -52,10 +58,13 @@ prepare: |
 
 execute: |
     LOOP="$(cat loop.txt)"
+
     echo "Run the snap-bootstrap tool"
-    /usr/lib/snapd/snap-bootstrap create-partitions --mount \
+    go get ../../../lib/create-partitions
+    create-partitions --mount \
```

Comment:
We could also test the creation of the TPM lockout autorization file and the policy update data file.